app/src/components/index.js

```diff
@@ -22,6 +24,12 @@ import { logout } from '../helpers/auth'
 
 import store from '../store';
 
+addLocaleData([...en, ...es]);
+
+const language = (navigator.languages && navigator.languages[0]) ||
+                     navigator.language ||
+                     navigator.userLanguage;
```

Comment:
This is a lot of fallbacks. Haha. Wonder if we should just slip "en" at the end to make sure a language setting doesn't slip through the cracks.

Also, this makes me think ahead to a potential feature that would allow a user to toggle the language. Perhaps on the user page you implemented.